dragonchain/webserver/lib/verifications.py

```diff
@@ -50,6 +52,42 @@ def query_verifications_v1(block_id: str, params: Optional[Dict[str, Any]] = Non
     return _get_verification_records(block_id, level)
 
 
+def query_subsequent_interchain_broadcasts_v1(block_id: str) -> List[Any]:
+    """Return the subsequent broadcasts to other L5 networks"""
+    _log.info(f"Getting subsequent L5 verifications for {block_id}")
+    results = []
+    l5_verifications = _get_verification_records(block_id, 5)
+    if len(l5_verifications) > 0:
+        timestamp = cast(List[Any], l5_verifications)[0]["header"]["timestamp"]
+        l5_nodes = redisearch._get_redisearch_index_client(redisearch.Indexes.verification.value).redis.smembers(redisearch.L5_NODES)
+        results = [_query_l5_verification(l5_dc_id.decode("utf-8"), timestamp) for l5_dc_id in l5_nodes]
+    return [storage.get_json_from_object(f"BLOCK/{x}") for x in results if x is not None]
+
+
+def _query_l5_verification(l5_dc_id: str, timestamp: str) -> str:
+    query_result = None
+    try:
+        query_result = redisearch.search(
+            index=redisearch.Indexes.verification.value,
+            query_str=f"@dc_id:{l5_dc_id} @timestamp:[{int(timestamp) + 1} +inf]",
+            only_id=True,
+            limit=1,
+            sort_by="timestamp",
+            sort_asc=True,
+        )
+    except redis.exceptions.ResponseError as e:
+        error_str = str(e)
```

Comment:
I don't think any of this additional error handling is relevant here. The index in your query is hardcoded and you're not passing user input into the query, so it should never have a syntax error. Probably just remove this entire try/except